Source/JavaScriptCore/b3/B3ReduceStrength.cpp

```diff
@@ -960,6 +961,16 @@ class ReduceStrength {
                     break;
 
                 default:
+                    // Turn this: Div(value, constant)
+                    // Into this: Shr(value, log2(constant))
+                    if (divisor64 > 0 && hasOneBitSet(divisor64)) {
+                        unsigned shiftAmount = WTF::fastLog2(static_cast<uint64_t>(divisor64));
```

Comment:
Nit: Wouldn't countTrailingZeros be slightly faster? Since reduce strength is already one of the slowest phases we might as well do what we can to speed it up.